lib/modules/platform/gitlab/readme.md

```diff
@@ -2,21 +2,25 @@
 
 ## Authentication
 
-You can authenticate Renovate to GitLab, with a Personal Access Token, or Group Access Token.
+You can authenticate Renovate to GitLab, with a [Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html), [Project Access Token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) or [Group Access Token](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html).
 
 To start, create either:
 
-- a [Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account
-- or a [Group Access Token](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html#bot-users-for-groups) for the bot account
-- or a [Deploy Token](https://docs.gitlab.com/ee/user/project/deploy_tokens/index.html) for the bot account
+- a [Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token) for the bot account
+- or a [Project Access Token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html#create-a-project-access-token) to the project Renovate will be working on
```

Comment:
I would like a warning here that a project token will only work with one project and is usually not the way people want to run renovate

Reply:
Sure, why not. Though I know of several monorepo projects that run Renovate as part of their CI/CD pipeline (schedule with rules).

Regarding warnings, I mentioned in the [discussion](https://github.com/renovatebot/renovate/discussions/30603) that this page overlaps with [[GitLab bot security](https://docs.renovatebot.com/gitlab-bot-security/#gitlab-bot-security)](https://docs.renovatebot.com/gitlab-bot-security/#gitlab-bot-security). For example, should we also mention that Project Access Tokens and Group Access Tokens require GitLab Premium or Ultimate tiers?

Reply:
I think it's ok to address these two docs separately or together if you prefer. I want to avoid people getting confused where they add a project token from their renovate runner repo and then don't understand why it can't discover any of their repos.